### PR TITLE
[New] Add color v0.11.3 w/ npm auto-update

### PR DESCRIPTION
**Pull request by maruilian11**
**https://www.npmjs.com/package/color**
**80 233 downloads in the last day
462 519 downloads in the last week
1 882 871 downloads in the last month**
**git repo url:** https://github.com/Qix-/color
**Watch 15
  Star 1,118
 Fork 90**
@Amomo could u help me check this PR for #8531 ? thank you!
**use `index*.js` rather than `index*` to avoid files like `index.html`**

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.
# Essential checklist
- [ ] I'm the author of this library
  - [ ] I would like to add link to the page of this library on CDNJS on website / readme
- [x] This lib was not found on cdnjs repo
- [x] No already exist / duplicated issue and PR 
- [x] The lib has notable popularity 
  - [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
  - [x] More than 500 downloads stats per month on npm registry 
- [x] Project has public repository on famous online hosting platform (or been hosted on npm)
# Auto-update checklist
- [x] Has valid tags for each versions (for git auto-update) 
- [x] Auto-update setup
- [x] Auto-update target/source is valid.
- [x] Auto-update filemap is correct.
# Git commit checklist
- [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
- [x] The parent of the commit(s) in the PR is not old than 3 days.
- [x] Pull request is sending from a non-master branch with meaningful name.
- [x] Separate unrelated changes into different commits.
- [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
- [x] Close corresponding issue in commit message
- [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

File: ajax/libs/color/0.11.3/index.js [hunk 1]
```diff
@@ -0,0 +1,450 @@
+/* MIT license */
+var clone = require('clone');
+var convert = require('color-convert');
+var string = require('color-string');
```
Comment (PeterDaveHello) [inline ajax/libs/color/0.11.3/index.js]: Does this work in a browser?

Comment (maruilian11) [inline ajax/libs/color/0.11.3/index.js]: I think it does.
according to the author said...
```
Using requireJS or something similar, yes -- to my knowledge none of the dependencies requires anything Node.js specific.
```

https://github.com/cdnjs/cdnjs/issues/8531#issuecomment-244194889

Comment (PeterDaveHello) [inline ajax/libs/color/0.11.3/index.js]: There is no requireJS here, this should be packaged first.

Comment (maruilian11) [inline ajax/libs/color/0.11.3/index.js]: sorry, what does you mean about being packged?

Comment (PeterDaveHello) [inline ajax/libs/color/0.11.3/index.js]: Google: AMD, CommonJS, and UMD
https://github.com/umdjs/umd

Comment (maruilian11) [inline ajax/libs/color/0.11.3/index.js]: wait for reply https://github.com/cdnjs/cdnjs/issues/8531#issuecomment-244551903

**Comment by PeterDaveHello** [comment]
Any updates here? Thanks!

**Comment by maruilian11** [comment]
ping the author already

**Comment by maruilian11** [comment]
@PeterDaveHello Since the author seems not going to provide packaged files, do we directly close this PR?

**Comment by PeterDaveHello** [comment]
The author says that? I didn't see it, did I miss it?

**Comment by sufuf3** [comment]
@PeterDaveHello @maruilian11 Here is the author's reply
https://github.com/cdnjs/cdnjs/issues/8531#issuecomment-326765668

Thank you.

**Comment by Qix-** [comment]
FWIW I think you should consider what I mentioned in my reply. If you are expecting package maintainers to babelify or otherwise transpile/build their libraries, you're going to _quickly_ become irrelevant as a CDN.

**Comment by PeterDaveHello** [comment]
Close this one first.

**Pull request closed without merging**